src/components/TopBar/components/StyledRouterLink.tsx

```diff
@@ -28,13 +45,18 @@ const StyleRouterLink: React.FC<StyleRouterLinkProps> = ({target, label, style,
         }}
         onClick={() => { 
           if (mobileMenu) {
-            setIsShow(!isShow)
+            setIsShow(!isShow);
           }
         }}
         style={mobileMenu ? {cursor: "pointer", width: "100%", textAlign: "left", display: "flex", flexDirection: "column"}
                           : {cursor: "pointer", width: "100%", textAlign: "left"}}
       >
-        <StyledLink exact activeClassName="active" style={style} to={target || ''}>{label}</StyledLink>
+        <StyledLink subactive={isActive} exact activeClassName="active" style={style} to={target || ''}>
+          {label}
+          { mobileMenu && (
+            <StyledSpan>🔻</StyledSpan>
```

Comment:
thats a very bright color for the design, for the moment we can use `➕` with adding `➖` depending on what state the sub menu is currently on. later we need to change this to be a png icon and not an emoji